node/api.go

```diff
@@ -56,7 +56,7 @@ func (api *PrivateAdminAPI) AddPeer(url string) (bool, error) {
 	if err != nil {
 		return false, fmt.Errorf("invalid enode: %v", err)
 	}
-	server.AddPeer(node)
+	server.AddPeer(node, "static")
```

Comment:
Optional Nit: What do you think about something slightly more specific, like "explicitStatic", or "rpcStatic"?

Reply:
Changed it to ExplicitStaticPurpose.